src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionSymbolService.cs

```diff
@@ -55,6 +61,16 @@ public async Task<(ISymbol, TextSpan)> GetSymbolAndBoundSpanAsync(Document docum
             return default;
         }
 
+        private static bool IsPartialMethodImplementation(ISymbol symbol)
+        {
+            if (symbol is IMethodSymbol methodSymbol)
+            {
+                return methodSymbol.PartialImplementationPart != null;
```

Comment:
would like to see this unified with SymbolEquivalentComparet.IsPartialMethodImplementationPart